TACC/Core-Portal
# FP-279: Ticket dialog should scroll to last message when opened

## Overview: ##

The ticket modal did not always appear to scroll to and open the last item when initially opened.

## PR Status: ##


## Related Jira tickets: ##

* [FP-279](https://jira.tacc.utexas.edu/browse/FP-279)

## Summary of Changes: ##
Updated the tickets saga to call the correct action when the modal is first opened

## Testing Steps: ##
1. Boot up local dev
2. Open a ticket, which brings up the ticket modal
3. The ticket modal should load up the ticket history items, scroll down to the latest/newest/bottom item and toggle it open.

## Notes: ##
If you need to confirm the scrolling works, create a test ticket (make sure your local dev is pointing to the wma portal rt ticket queue), and then add a bunch of replies from either RT or the ticket modal, then open the ticket modal again from the dashboard and you should see it scroll.
